tests/test_views.py

```diff
@@ -1368,11 +1411,24 @@ def test_get_course_enrollment_page_with_no_course_effort(
         assert response.status_code == 200
         course_modes = [
             {
-                'mode': 'enroll',
-                'title': 'Enroll',
-                'original_price': '$200',
-                'final_price': '$190',
-                'description': 'Discount provided by Starfleet Academy',
+                "mode": "professional",
+                "title": "Professional Track",
+                "original_price": "$100",
+                "final_price": "$50",
+                "description": "Earn a verified certificate!",
+                "premium": True,
+
+            }
+        ]
+        premium_modes = [
+            {
+                "mode": "professional",
+                "title": "Professional Track",
+                "original_price": "$100",
+                "final_price": "$50",
+                "description": "Earn a verified certificate!",
+                "premium": True,
+
```

Comment:
@replaceafill Same comment here: Please remove empty lines and extract dict into a variable. At this point I think that defining the dict at class level (or turning it into a file-level constant) might be most appropriate.